src/node/ledgersecrets.h

```diff
@@ -43,68 +43,64 @@ namespace ccf
     {}
   };
 
-  class LedgerSecrets
+  struct LedgerSecrets
```

Comment:
Why is this changed from class to struct? We should add an access specifier for `secrets_map` and `operator==`, to make it clear whether they're intended to be `public` or `private`.

Reply:
Good spot. I've reverted this to a `class`. The `secrets_map` is `public` so that we can serialised it in the Join RPC response.